sphinx/util/console.py

```diff
@@ -93,7 +99,7 @@ def escseq(name: str) -> str:
 
 
 def strip_colors(s: str) -> str:
-    return re.compile('\x1b.*?m').sub('', s)
+    return _ansi_color_re.sub('', s)
```

Comment:
@jayaddison the regex was the same in `strip_escseq` (and color codes are essentially characterized by the 'm' suffix).